src/components/Inventory/Inventory.js

```diff
@@ -61,6 +62,7 @@ const itemTypeCategoryMap = Object.freeze({
   [SPRINKLER]: FIELD_TOOLS,
   [STONE]: MINED_RESOURCES,
   [TOOL_UPGRADE]: UPGRADES,
+  [WEEDS]: CROPS,
```

Comment:
Rather than categorizing Weeds under Crops, what do you think about introducing a new inventory category to display them under? Perhaps "Foraged Items?"

Reply:
i really like this because it's a perfect spot for mushrooms to go too!

Reply:
Ooooh that would be an amazing addition. I love the idea! 🍄